app/src/processing/app/Base.java

```diff
@@ -118,6 +118,11 @@
   Editor activeEditor;
 
   private static JMenu boardMenu;
+  private static ButtonGroup boardsButtonGroup;
+  private static ButtonGroup recentBoardsButtonGroup;
```

Comment:
Does `recentBoardsButtonGroup` need to be a ButtonGroup? AFAIU, a ButtonGroup is intended to group radio buttons so only one of them can be selected at once, but all recent-boards radiobuttons are also added to `boardsButtonGroup`, so they are already mutually exclusive. It seems you are using `recentBoardsButtonGroup` just to keep track of all recent-boards menu items, and then it is probably better to use a plain LinkedList or other collection?